jni_bindings/src/lib.rs

```diff
@@ -278,176 +341,192 @@ fn from_attr(attr: &dom::Attribute) -> Attribute {
 }
 
 #[no_mangle]
-pub extern "system" fn Java_org_phoenixframework_liveview_lib_Document_get_1node_1leaf_1string<
+pub unsafe extern "system" fn Java_org_phoenixframework_liveview_lib_Document_get_1node_1leaf_1string<
     'local,
 >(
-    env: JNIEnv<'local>,
+    mut env: JNIEnv<'local>,
     _: JClass<'local>,
+    // non-null raw pointer to Document
```

Comment:
The `non-null` can't be true or there would be no need for the `this.is_null()` check below.
```suggestion
    // raw pointer to Document
```